seed/InsertCampaigns.js

```diff
@@ -0,0 +1,24 @@
+conn = new Mongo();
+db = conn.getDB("ethics-api");
+
+for(var i = 1; i <= 3; i ++) {
+  var json = {
+    _id: i.toString(),

```

Comment:
I'm probably going to change these after I merge it in to use ObjectID's by the way. That's what we should be using as IDs in MongoDB unless there's a good reason to use something else. :)